.buildkite/pipeline.yml

```diff
@@ -2,7 +2,7 @@
 
 env:
   SETUP_GVM_VERSION: 'v0.5.0'
-  GO_VERSION_CHOCO: "1.20.5"
+  GO_VERSION_CHOCO: "1.21.12"
```

Comment:
We need to get the value from the content of https://github.com/elastic/elastic-agent-autodiscover/blob/main/.go-version.

If this YAML block doesn't support it, this can be done in the individual scripts. Elastic Agent installs GVM which knows how to read the .go-version file to deal with this. https://github.com/elastic/elastic-agent/blob/4d074b9ff02433e0c3a70f5e31d55db52fbd9b9d/.ci/scripts/install-go.sh

Reply:
Unfortunately it seems like chocolatey does not offer all the available go version for windows - https://community.chocolatey.org/packages/golang#versionhistory 1.21.7 seems to be the last one in the 1.21.x serie

Reply:
Move this repository to Go 1.22 then

Reply:
1.22.5 and running